CLI/Repl.cpp

```diff
@@ -253,6 +253,41 @@ static void completeRepl(lua_State* L, const char* editBuffer, std::vector<std::
     completeIndexer(L, editBuffer, start, completions);
 }
 
+#ifdef LUAU_WEB_REPL
+extern "C"
+{
+    // Luau errors are exceptions (see luaD_throw) which cannot be directly
```

Comment:
What are the cases when an exception escapes? It should always be caught at runCode level in theory.

Reply:
When testing it was all of them. I'm not too sure why just yet, I can do some more digging. I think it might be something to do with Emscripten's compiler and how it handles exceptions.

Reply:
That's definitely a problem since it will likely mean `pcall` doesn't work among other things...

Reply:
You're right. 
![image](https://user-images.githubusercontent.com/28831011/140780664-639bcf95-7010-4b04-b8e1-8975c7fef695.png)

Reply:
Alright, I think I've figured out a solution. Going to do some tests and get back to you.

Reply:
This is now resolved.